clusterman/aws/markets.py

```diff
@@ -417,8 +417,8 @@ def fetch_instance_type_from_aws(instance_type: str) -> InstanceResources:
     return InstanceResources(vcpu_count, mem_size, disk_size, gpu_size)
 
 
-def get_instance_type(instance_type: str) -> InstanceResources:
-    cluster_name = os.environ.get("CMAN_CLUSTER", None)
+def get_instance_type(instance_type: str, cluster: str = None) -> InstanceResources:
+    cluster_name = os.environ.get("PAASTA_CLUSTER", cluster)
```

Comment:
I'd suggest to just make the function argument compulsory and authoritative to avoid surprises

```suggestion
def get_instance_type(instance_type: str, cluster_name: str) -> InstanceResources:
```

Reply:
making `cluster` var non-default is tricky, a lot for mock test depends on this and InstanceMarket.
@piax93